aptos-move/e2e-move-tests/tests/code_publishing.rs

```diff
@@ -302,3 +328,40 @@ fn code_publishing_using_resource_account() {
     );
     assert_success!(result);
 }
+
+#[rstest]
+#[case(vec![])]
+#[case(vec![CODE_DEPENDENCY_CHECK])]
+fn code_publishing_faked_dependency(#[case] features: Vec<u64>) {
```

Comment:
Correct me if I'm wrong, this seems like the only relevant test to add rstest to. Are the other ones just for broader validation?

Reply:
You are right. Initially I thought we should do this for every test, but this seems to be waste of CI time. So I now apply both flag combinations only to the basic test and to this one. 

Still keeping the `#[rstest]` now only with one `#[case]` for the situation we want to add more combos.